### PR TITLE
feat: archive-note

### PR DESCRIPTION
**Pull request by BelideNikhil**
### features

+ archive note
+ un-archive note
+ edit note in archive

### CODE REVIEW

**Review by poornima-alugubelly — APPROVED** [review]
Nothing to comment the whole code is very clean good work 👍

**Review by Kavita539 — APPROVED** [review]
- As always  your code looks clean and good.
-  Love the UI, clean and minimal.

**Review by KaustubhM1997 — APPROVED** [review]
Everything looks clean. Good to merge!

File: src/Components/ArchivedList/ArchivedList.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import "./ArchivedList.css";
+import { useNotes } from "../../Context";
+import { NoteCard } from "../NoteCard/NoteCard";
+export function ArchivedList() {
+    const {
+        notesState: { archivedList },
+    } = useNotes();
+    console.log(archivedList);
```
Comment (srishti-maurya) [inline src/Components/ArchivedList/ArchivedList.jsx:8]: you may remove the `console.log`

**Pull request merged**